src/Storage/FileStorage.php

```diff
@@ -62,16 +62,17 @@ public function getData(): array
 
     public function flush(): void
     {
+        $basePath = $this->path . '/' . date('Y-m-d H:i:s') . '/' . $this->idGenerator->getId() . '/';
```

Comment:
I think, time format is not needed here
```suggestion
        $basePath = $this->path . '/' . date('Y-m-d') . '/' . $this->idGenerator->getId() . '/';
```